frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions.unit.spec.tsx

```diff
@@ -60,7 +64,17 @@ function setup() {
 
 describe("ExpressionEditorSuggestions", () => {
   test("suggestions items should show column info icon", async () => {
-    setup();
+    setup({ source: "[" });
+
+    await screen.findAllByTestId("expression-suggestions-list-item");
```

Comment:
if it's needed to wait for element to appear, it's better to clarify it

either by using expect or by using waitFor